src/kiwixapp.cpp

```diff
@@ -281,84 +281,72 @@ bool KiwixApp::isCurrentArticleBookmarked()
     return false;
 }
 
-#define CREATE_ACTION_ICON(ID, ICON, TEXT) \
-    mpa_actions[ID] = new QAction(QIcon(":/icons/" ICON ".svg"), TEXT)
-#define CREATE_ACTION(ID, TEXT) \
-    mpa_actions[ID] = new QAction(TEXT)
-#define SET_SHORTCUT(ID, SHORTCUT) mpa_actions[ID]->setShortcut(SHORTCUT)
+#define CREATE_ACTION_ICON_SHORTCUT(ID, ICON, TEXT, SHORTCUT) \
+    mpa_actions[ID] = new QAction(QIcon(":/icons/" ICON ".svg"), TEXT); \
+    mpa_actions[ID]->setShortcut(SHORTCUT); \
+    mpa_actions[ID]->setToolTip(TEXT + " (" +  QKeySequence(SHORTCUT).toString() + ")" ); \
+    mpa_actions[ID]->setIconVisibleInMenu(false)
```

Comment:
We probably don't want to set all icons invisible in menu.
We were setting the icon invisible only for closeTabAction.
Now we are setting this for all actions.

Reply:
Yeah, I kind of assumed all should be invisible in menu, will fix this thanks!